pkg/service/did/ion.go

```diff
@@ -47,6 +47,8 @@ type ionHandler struct {
 	keyStore *keystore.Service
 }
 
+var _ MethodHandler = (*ionHandler)(nil)
```

Comment:
Maybe put a comment that CHAD told me about:
This line doesn't actually do anything at runtime; it's purely for compile-time checking. If ionHandler doesn't implement all methods required by MethodHandler, you'll get a compile-time error.